MANUAL.txt

```diff
@@ -2518,6 +2518,15 @@ For headerless tables, the colons go on the top line instead:
     | Right         | Left          | Centered           |
     +---------------+---------------+--------------------+
 
+##### Grid Table Limitations #####
+
+Pandoc does not support grid tables with row spans (variable number
```

Comment:
Actually, I think the usual name for this is "column spans" (since the cell spans multiple columns).
Row spans are cells that span multiple rows. Actually, pandoc doesn't support either.